.github/workflows/codeql-analysis.yml

```diff
@@ -74,7 +74,7 @@ jobs:
 
       - name: Set up Maven
         run:
-          mvn -e -B -V org.apache.maven.plugins:maven-wrapper-plugin:3.1.0:wrapper "-Dmaven=3.9.4"
+          mvn -e -B -V org.apache.maven.plugins:maven-wrapper-plugin:3.1.0:wrapper "-Dmaven=3.9.6"
```

Comment:
Is this necessary for the infinispan update?

Reply:
not really but doesn;t hurt to use same version as version used in Jenkins